launch_testing_ros/test/examples/wait_for_topic_launch_test.py

```diff
@@ -57,11 +59,16 @@ def test_topics_successful(self, count):
             """All the supplied topics should be read successfully."""
             topic_list = [('chatter_' + str(i), String) for i in range(count)]
             expected_topics = {'chatter_' + str(i) for i in range(count)}
+            message_pattern = re.compile(r'Hello World: \d+')
 
             # Method 1 : Using the magic methods and 'with' keyword
-            with WaitForTopics(topic_list, timeout=2.0) as wait_for_node_object_1:
+            with WaitForTopics(topic_list, timeout=2.0,
+                               messages_received_buffer_length=10) as wait_for_node_object_1:
```

Comment:
Also, kind of meta, but do you need to pass the custom buffer length here? Is it really exercising anything?

Reply:
In the original implementation, I was not using deque and just storing every message.
Then @adityapande-1995 suggested using `deque` and letting the user choose the buffer length, and so I did.
So this question is better answered by @adityapande-1995 .